fax/constrained/constrained.py

```diff
@@ -304,7 +305,8 @@ def _equality_constraints(variables):
 
     @jit
     def update(i, opt_state):
-        grads = grad(lagrangian, (0, 1))(*get_params(opt_state))
+        grad_fn = grad(lagrangian, (0, 1))
```

Comment:
Aesthetical?

Reply:
it allows for exception messages to be more detailed, both the `grad` and grad evaluation are complex functions.
reverting for the same reasoning as above